loader/trampoline.c

```diff
@@ -73,10 +73,14 @@ LOADER_EXPORT VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL vkGetInstanceProcAddr(VkI
             // Old behavior is to return a global entrypoint regardless of the value of the instance handle.
             // Use new behavior if: The instance is valid and the minor version of the instance is greater than 1.2, which
             // was when the new behavior was added. (eg, it is enforced in the next minor version of vulkan, which will be 1.3)
+            // Maintain this behavior for vulkansc version 1.1.
 
             // First check if instance is valid - loader_get_instance() returns NULL if it isn't.
             struct loader_instance *ptr_instance = loader_get_instance(instance);
-            if (ptr_instance != NULL && (ptr_instance->app_api_minor_version > 2)) {
+            if (ptr_instance != NULL &&
+               (((ptr_instance->app_api_variant_version == 0) && (ptr_instance->app_api_minor_version > 2)) ||
+               ((ptr_instance->app_api_variant_version == VKSC_API_VARIANT) && (ptr_instance->app_api_minor_version > 0))))
```

Comment:
VKSC_API_VARIANT isn't defined in Vulkan mode, this is the cause of many CI failures.

Reply:
ah, ok I guess I can just use "== 1". I figured it would be better to use a define when possible but forgot that this define isn't in the standard vulkan headers.